3.3/examples/django/application-template-stibuild.json

```diff
@@ -0,0 +1,282 @@
+{
+  "apiVersion": "v1beta1",
+  "items": [
+    {
+      "apiVersion": "v1beta1",
+      "containerPort": 8080,
+      "id": "frontend",
+      "kind": "Service",
+      "port": 5432,
```

Comment:
I know we have that weird port in our sample-app example, but maybe changing it here to something different.